spec/factories/physical_storage.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :physical_storage,
+          :class => "ManageIQ::Providers::Autosde::StorageManager::PhysicalStorage" do
+    name { "test_physical_storage" }
```

Comment:
If a name required (e.g. has a `validates :name, :presence => true`) then you would need to have something like this here in the factory but it isn't required.
What we shouldn't do is have a constant name, if you need to do this you can use a sequence to keep the name unique

Reply:
good to know 👍